client/web/package.json

```diff
@@ -59,15 +59,16 @@
     "webpack-merge": "^5.7.3"
   },
   "dependencies": {
-    "@genesislcap/foundation-comms": "^6.0.0",
-    "@genesislcap/foundation-entity-management": "^6.0.0",
-    "@genesislcap/foundation-header": "^6.0.0",
-    "@genesislcap/foundation-login": "^6.0.0",
-    "@genesislcap/foundation-ui": "^6.0.0",
-    "@genesislcap/foundation-utils": "^6.0.0",
-    "@genesislcap/foundation-zero": "^6.0.0",
-    "@genesislcap/foundation-zero-grid-pro": "^6.0.0",
-    "@genesislcap/g2plot-chart": "^6.0.0",
+    "@genesislcap/alpha-design-system": "10.6.0",
+    "@genesislcap/foundation-comms": "8.0.2-alpha-2368811.0+2368811",
+    "@genesislcap/foundation-entity-management": "10.6.0",
+    "@genesislcap/foundation-header": "10.6.0",
+    "@genesislcap/foundation-login": "8.0.2-alpha-2368811.0+2368811",
+    "@genesislcap/foundation-ui": "10.6.0",
+    "@genesislcap/foundation-utils": "10.6.0",
+    "@genesislcap/foundation-zero": "10.6.0",
+    "@genesislcap/foundation-zero-grid-pro": "10.6.0",
+    "@genesislcap/g2plot-chart": "10.6.0",
```

Comment:
We should add configurator package here too and add a NPM task for launching it. We could then mention it in the 'your app has been created` message at the end.